openedx_caliper_tracking/settings.py

```diff
@@ -16,4 +16,4 @@
 CALIPER_AUTH = {
     'CALIPER_TESTS_API_URL': '',
```

Comment:
Shouldn't these be loaded from the settings module?

Reply:
yes that can be done, but its a unit test. So i think we shouldn't.

Reply:
You should. There is a test config present in Open edX. Please add code and documentation to get these vars from the config.

Reply:
Okay, got it. thanks.